frontend/apps/core-app/eas.json

```diff
@@ -0,0 +1,39 @@
+{
+  "cli": {
+    "version": ">= 0.43.0"
+  },
+  "build": {
+    "development": {
+      "distribution": "internal",
+      "releaseChannel": "development",
+      "env": {
+        "REACT_APP_OIDC_ISSUER": "",
```

Comment:
aren't those currently generated to contain the local tunnels?

Reply:
When we run this locally it will use the `.env` file. This is for when we build an `apk` and want it pointing at a deployed service. These should be populated, but I don't have any values for them yet.